### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by rsf92**
I've translated a few strings into spanish, along with correcting a typo I saw on the original english file.

### CODE REVIEW

**Comment by Vezzra** [comment]
@Oberlus, I think you're the only one who can review this...

File: default/stringtables/es.txt
Comment (Oberlus) [inline default/stringtables/es.txt]: Corrección: "desbloque**o**"

File: default/stringtables/es.txt
Comment (Oberlus) [inline default/stringtables/es.txt]: Salvo que haya un problema con la anchura de la palabra, prefiero "Desperdiciada" en lugar de "Perdida".

File: default/stringtables/es.txt
Comment (Oberlus) [inline default/stringtables/es.txt]: Salvo que haya un problema con la anchura de la palabra, prefiero "Desperdiciada" en lugar de "Perdida".

**Review by Oberlus — CHANGES_REQUESTED** [review]

File: default/stringtables/es.txt
Comment (rsf92) [reply default/stringtables/es.txt]: De acuerdo, mirando el fichero, he visto que es común poner dos espacios después de un . o un ?, ¿es esto otro error o tiene algún sentido dentro del código? Por meter también esos cambios si hace falta

**Comment by rsf92** [comment]
Vale, tengo los cambios, en principio, pero voy a esperar a que me contestes sobre el par de cosas que he comentado para subirlos

File: default/stringtables/es.txt
Comment (Oberlus) [reply default/stringtables/es.txt]: I think that if "Títulos de ventana" (Windows titles) doesn't fit in the standard window size, you could use instead "Títulos" (Titles) rather than "Ventanas" (Windows). What do you think @geoffthemedio @Vezzra ?

File: default/stringtables/es.txt
Comment (Oberlus) [reply default/stringtables/es.txt]: I think there should be no double spaces. If there are double spaces (in the English or the Spanish stringtables) after each period or interrogation, I don't know if that is because of some FOCS requirement (I don't think so) or for some aesthetical reason. I would remove them all but I'm unsure. @geoffthemedio ?

Comment (geoffthemedio) [reply default/stringtables/es.txt]: I'd remove all double spaces that aren't there for specific reasons. Different people wrote bits of stringtable over many years, so there are probably some cases where they are present, though.

File: default/stringtables/es.txt
Comment (geoffthemedio) [reply default/stringtables/es.txt]: "Titles" seems more helpful than "Windows" in this context

**Comment by rsf92** [comment]
Okay, I pushed the changes + some corrections detected by the workflow - which i guess should be run again to make sure everything is ok now, awaiting further feedback

File: default/stringtables/es.txt
Comment (Oberlus) [inline default/stringtables/es.txt]: Corrección: **Espacio habitable** en planetas enanos
(Y lo mismo para el resto de tamaños).

**Review by Oberlus — CHANGES_REQUESTED** [review]

**Comment by Oberlus** [comment]
> Okay, I pushed the changes + some corrections detected by the workflow - which i guess should be run again to make sure everything is ok now, awaiting further feedback

I missed one more thing (commented above), but once it's fixed this seems good to merge.

File: default/stringtables/es.txt
Comment (rsf92) [reply default/stringtables/es.txt]: Nuevamente el problema es que el texto de la traducción literal no cabía en la ventana, por lo que cambié a hábitats, no se si hay alguna otra opción mejor...

Comment (Oberlus) [reply default/stringtables/es.txt]: En ese caso, "Habitabilidad de planetas enanos". Si no caben ninguna de las dos, la versión completa, y espero que redimensionando la ventana se vea entero. In extremis, abrir un issue para que traten el tema de las ventanas demasiado pequeñas para algunos idiomas.

Comment (rsf92) [reply default/stringtables/es.txt]: De acuerdo, ya he subido el cambio, va a ser necesario redimensionar la ventana, por eso he puesto también al final lo de títlulo de ventanas.

EDIT: He mirado lo de crear issues y por no liarla, ¿Te importaría crearla tu @Oberlus?, adjunto capturas de pantalla de algunos strings que no salen completos

Captura de pantalla de 2021-09-14 18-14-14
Captura de pantalla de 2021-09-14 18-14-49
Captura de pantalla de 2021-09-14 18-21-50

Comment (Oberlus) [reply default/stringtables/es.txt]: Entendido. Gracias por los screenshots. Yo creo la issue (primero abriré hilo en el foro).

**Comment by Cjkjvfnby** [comment]
@rsf92 , I've restarted CI cheks. All except `String tables checks / lint-string-tables (3.9) (pull_request) ` are irrelevant to your changes, just ignore them.


PS. If it's convenient for you you could run string-tables tool from the console.  
https://github.com/freeorion/freeorion/blob/master/check/st-tool.py
```bash
cd freeorion  # repo folder
./check/st-tool.py --help
```

**Comment by rsf92** [comment]
@Cjkjvfnby  i understand i did okay and it simply does not print anything if there's no error, right?
Captura de pantalla de 2021-09-15 07-32-09

**Comment by Cjkjvfnby** [comment]
> @Cjkjvfnby i understand i did okay and it simply does not print anything if there's no error, right?

Yes.   Also, I recommend adding the reference  ```--reference default/stringtables/en.txt```.  

This is CI job definition: https://github.com/freeorion/freeorion/blob/master/.github/workflows/string-tables-check.yml#L26

**Comment by rsf92** [comment]
Things seem fine @Cjkjvfnby 
Captura de pantalla de 2021-09-16 07-31-14

File: default/stringtables/es.txt
Comment (Vezzra) [reply default/stringtables/es.txt]: Agree with @geoffthemedio...

Comment (rsf92) [reply default/stringtables/es.txt]: I ended up undoing the change, it's now Window Titles (Tñitulos de Ventana)

Comment (Vezzra) [reply default/stringtables/es.txt]: Ok!

**Comment by Cjkjvfnby** [comment]
@rsf92 , just a small advice for the future PR's:  Try to keep them small 10-20 entries. This will reduce review time (it's much easier to find 5 minutes for review than 1 hour). 

If you are familiar with git, you could create a couple of them simultaneously and resolve conflicts after merges, if not you can do it one by one.

Feel free to ask me about git question.

Also any feedback about st-tool is welcome. If you feel that some checks or features could be useful, feel free to send me a request.

**Comment by rsf92** [comment]
> @rsf92 , just a small advice for the future PR's: Try to keep them small 10-20 entries. This will reduce review time (it's much easier to find 5 minutes for review than 1 hour).
> 
> If you are familiar with git, you could create a couple of them simultaneously and resolve conflicts after merges, if not you can do it one by one.
> 
> Feel free to ask me about git question.
> 
> Also any feedback about st-tool is welcome. If you feel that some checks or features could be useful, feel free to send me a request.

Thanks for your advice, yeah i thought maybe it was too big of a merge to add, i will do them smaller from the time being.

I know some git, yeah, no worries

On the st-tool, i'll have a closer look at it just in case, but I think it would be helpful to have it be a bit more verbosed, or add this option as a command line parameter

**Comment by rsf92** [comment]
HI @Cjkjvfnby and @Oberlus, i've received no further feedback on this PR neither has it been merged. I understand it might be because of lack of time on your part, so this is just a quick reminder in case there's something else i need to do

**Comment by Oberlus** [comment]
> HI @Cjkjvfnby and @Oberlus, i've received no further feedback on this PR neither has it been merged. 

Regarding translation correctness, you've got my green light for the merge, if @Cjkjvfnby is OK with it too.

cc @geoffthemedio

**Comment by geoffthemedio** [comment]
I think @Cjkjvfnby 's comments were about the PR process, not really the content of this one in particular.

**Pull request merged**
